epicapp/epicappAPI/models.py

```diff
@@ -8,7 +8,7 @@
 
 
 class Author(AbstractUser):
-    type = models.CharField(max_length=255, default="author", editable=False)
+    type = "author"
```

Comment:
btw, i changed this to models.CharField(max_length=255, default="author", editable=False) cause it looks more professional and cooler, but the issues that you get when runnung the server is only because you have to migrate (probably reset) the db before its good.